### DISCUSSION TITLE
Implement YIELD function

### DISCUSSION THREAD

**Discussion by tansukhrathod** [PHPOffice/PhpSpreadsheet#1838] [General]
Hello,

Hope doing well, When we can expect the YIELD function into the CATEGORY_FINANCIAL

This is the most important function in financial calculation of bonds. 
At the time we are facing trouble to calculate YIELD

So any update regarding this.

Thanks,
Tansukh Rathod

**Comment by MarkBaker** [comment]
The best that I can offer is "probably sometime this year".

Office 365 has introduced some significant changes to datatypes (objects in Excel); a series of additional array-based functions; and lambdas... all of these will need to be addressed this year... but I will try to ensure that the remaining Excel functions (apart from the cube functions) have basic implementations as a part of that series of changes.
